vehicle/OVMS.V3/components/vehicle_nissanleaf/src/vehicle_nissanleaf.cpp

```diff
@@ -458,7 +434,12 @@ void OvmsVehicleNissanLeaf::IncomingFrameCan1(CAN_frame_t* p_frame)
         }
       break;
     case 0x5c0:
-      if (d[0] == 0x40)
+      /* Another "ambient" temperature, but this one reacts to outside changes
```

Comment:
Change this comment to.

Battery Temperature as reported by the LBC. Effectively has only 7-bit precision, as the bottom bit is always 0.